src/sentry/api/paginator.py

```
@@ -811,3 +815,98 @@ def get_result(self, limit: int, cursor: Cursor | None = None):
+
+    def get_result(self, limit=100, cursor=None, count_hits=False, known_hits=None, max_hits=None):
+        # Enhanced cursor handling with advanced boundary processing
+        if cursor is None:
+            cursor = Cursor(0, 0, 0)
+
+        limit = min(limit, self.max_limit)
+
+        if cursor.value:
+            cursor_value = self.value_from_cursor(cursor)
+        else:
+            cursor_value = 0
```

Ruff (0.12.2)

Use ternary operator `cursor_value = self.value_from_cursor(cursor) if cursor.value else 0` instead of `if`-`else`-block

Replace `if`-`else`-block with `cursor_value = self.value_from_cursor(cursor) if cursor.value else 0`

(SIM108)